src/strongbox/catalogue.clj

```diff
@@ -47,13 +47,18 @@
 (defn-spec expand-summary (s/or :ok :addon/expanded, :error nil?)
   "fetches updates from the addon host for the given `addon` and `game-track`.
   when `strict?` is `false` and an addon fails to match for the given `game-track`, other game tracks will be checked.
+  the strategy is to assume the next-best game tracks are the ones updated most recently but no more than one classic release removed.
```

Comment:
> but no more than one classic release removed

hrm, so by that logic `:retail` should only have `:classic` available to it? and `classic-tbc` would have `:classic` and `classic-wrath` available? There just isn't much rhyme or reason here. Perhaps we shouldn't consider any releases 'above' it, so `:classic-tbc` wouldn't consider `:classic-wrath` or `:classic-cata`, just `:classic`